core/templates/css/oppia.css

```diff
@@ -2527,8 +2527,570 @@ oppia-parameter {
+  color: #D10B23;
+}
+
+.improvements-tab-navigation {
+  align-items: stretch;
+  border-left: 2px solid rgba(0,0,0,0.06);
+  display: flex;
+  flex-direction: column;
+  flex-grow: 0;
+  height: 100%;
+  justify-content: stretch;
+  margin: 0;
+  padding: 0;
+  width: 212px;
+}
+
+.improvements-tab-navigation-item {
+  align-items: center;
+  background-color: #F5F5F5;
+  display: flex;
+  flex: 1 1 auto;
+  list-style: none;
+}
+
+.improvements-tab-navigation-item + .improvements-tab-navigation-item {
+  border-top: 2px solid rgba(0,0,0,0.06);
+}
+
+.improvements-tab-navigation-item a.task-scope {
+  color: #00829B;
+  display: inline-flex;
+  font-size: 14px;
+  letter-spacing: 0.23px;
+  line-height: 16px;
+}
+
+.improvements-tab-navigation-item a.task-scope:hover {
+  text-decoration: none;
+}
+
+.improvements-tab-navigation-item span.dot-healthy {
+  background-color: #419848;
+  border-radius: 50%;
+  display: inline-block;
+  height: 8px;
+  margin: 4px 6px 4px 12px;
+  width: 8px;
+}
+
+.improvements-tab-navigation-item span.dot-warning {
+  background-color: #F2A900;
+  border-radius: 50%;
+  display: inline-block;
+  height: 8px;
+  margin: 4px 6px 4px 12px;
+  width: 8px;
+}
+
+.improvements-tab-navigation-item span.dot-critical {
+  background-color: #D10B23;
+  border-radius: 50%;
+  display: inline-block;
+  height: 8px;
+  margin: 4px 6px 4px 12px;
+  width: 8px;
+}
+
+.improvements-tab-navigation-item a.task-scope strong.number {
+  display: inline-block;
+  font-weight: bold;
+  width: 25px;
+}
+
+.improvements-tab-navigation-item a.task-scope span.content {
+  display: inline-block;
+  width: 129px;
+}
+
+.improvements-tab-navigation-item a.task-scope i.chevron-right {
+  display: inline-block;
+  font-size: 14px;
+  height: 9px;
+  width: 6px;
+}
+
+.improvements-tab-body {
+  background-color: #EEEEEE;
+  margin: 66px auto;
+  max-width: 1238px;
+  padding: 0;
+  width: 100%;
+}
+
+.improvements-tab-body hr {
+  margin: 0;
+}
+
+.improvements-level {
+  margin-bottom: 31px;
+  margin-top: 67px;
+}
+
+.improvements-level .level-header {
+  color: rgba(0, 0, 0, 0.87);
+  font-size: 20px;
+  font-style: normal;
+  font-weight: 500;
+  letter-spacing: 0.333333px;
+  line-height: 23px;
+  margin-bottom: 17px;
+  text-transform: uppercase;
+}
+
+.improvements-level .state-groups {
+  margin-left: 8px;
+  margin-top: 48px;
+}
+
+.state-groups .state-header {
+  display: flex;
+  margin-bottom: 19px;
+  margin-right: 362px;
+}
+
+.state-header .collapse-arrow {
+  color: #000;
+  flex-grow: 0;
+  height: 100%;
+  width: 40px;
+}
+
+.collapse-arrow .collapse-arrow-button {
+  font-size: 24px;
+  font-weight: bold;
+  text-align: center;
+  vertical-align: middle;
+  width: 24px;
+}
+
+.state-header .state-details {
+  display: flex;
+  flex-direction: column;
+  flex-grow: 1;
+}
+
+.state-details .state-link {
+  flex-grow: 0;
+  height: 21px;
+}
+
+.state-link .state-link-a {
+  color: #004C76 !important;
```

Comment:
Don't use "important" in CSS. If you need to, use more qualifiers in the rule selector.